### DISCUSSION TITLE
Force radv vulkan driver?

### DISCUSSION THREAD

**Discussion by zany130** [catsout/wallpaper-engine-kde-plugin#225] [Q&A] [answered]
I need `amdvlk` for a certain game (trails of cold steal 4)  but `amdvlk` break's scene wallpapers if the plugin uses it. Maybe and option could be added for selecting a specific vulkan driver

**Comment by catsout** [comment]
How do you switch the vulkan driver on a game?  
Setting environment variable?

**Reply by zany130** [reply]
yeah with `VK_ICD_FILENAMES` as mentioned here https://wiki.archlinux.org/title/Vulkan#Switching_between_AMD_drivers don't know if it would be possible to add an option that sets this env for the plugin though.

**Comment by catsout** [comment, marked as answer]
```sh
mkdir -p ~/.config/systemd/user/plasma-plasmashell.service.d
systemctl --user edit plasma-plasmashell.service
```
Add following:  
```text
[Service]
Environment=AMD_VULKAN_ICD=RADV
```
Then:  
`systemctl --user restart plasma-plasmashell.service`
